web-client/public/js/setup-handlers.js

```diff
@@ -423,6 +429,48 @@ export const setupHandlers = grnState => {
         updateApp(grnState);
     });
 
+    $(SPECIES_BUTTON_YEAST).click(function () {
+        var selection = $(SPECIES_BUTTON_YEAST).attr("id");
+        grnState.genePageData.identified = false;
+        identifySpeciesMenu(selection);
+        updateApp(grnState);
+    });
+
+    $(SPECIES_BUTTON_MOUSE).click(function () {
+        var selection = $(SPECIES_BUTTON_MOUSE).attr("id");
+        grnState.genePageData.identified = false;
+        identifySpeciesMenu(selection);
+        updateApp(grnState);
+    });
+
+    $(SPECIES_BUTTON_HUMAN).click(function () {
+        var selection = $(SPECIES_BUTTON_HUMAN).attr("id");
+        grnState.genePageData.identified = false;
+        identifySpeciesMenu(selection);
+        updateApp(grnState);
+    });
+
+    $(SPECIES_BUTTON_NEMATODE).click(function () {
+        var selection = $(SPECIES_BUTTON_NEMATODE).attr("id");
+        grnState.genePageData.identified = false;
+        identifySpeciesMenu(selection);
+        updateApp(grnState);
+    });
+
+    $(SPECIES_BUTTON_CRESS).click(function () {
+        var selection = $(SPECIES_BUTTON_CRESS).attr("id");
+        grnState.genePageData.identified = false;
+        identifySpeciesMenu(selection);
+        updateApp(grnState);
+    });
+
+    $(SPECIES_BUTTON_FLY).click(function () {
+        var selection = $(SPECIES_BUTTON_FLY).attr("id");
+        grnState.genePageData.identified = false;
+        identifySpeciesMenu(selection);
+        updateApp(grnState);
+    });
+
```

Comment:
These functions all have the same general structure, varying only by certain values—ripe for a parameterized helper function. But this consolidation can happen later.